src/extensions/averagewaveforms/AverageWaveformsView/AverageWaveformsView.tsx

```diff
@@ -1,34 +1,92 @@
-import React, { FunctionComponent, useMemo } from 'react';
+import React, { FunctionComponent, useCallback, useEffect, useMemo, useState } from 'react';
+import { FaArrowDown, FaArrowUp } from 'react-icons/fa';
+import sizeMe, { SizeMeProps } from 'react-sizeme';
 import SortingUnitPlotGrid from '../../common/SortingUnitPlotGrid';
+import Splitter from '../../common/Splitter';
+import { ActionItem, DividerItem } from '../../common/Toolbars';
 import { SortingViewProps } from "../../extensionInterface";
+import AverageWaveformsToolbar from './AverageWaveformsToolbar';
 import AverageWaveformView2 from './AverageWaveformView2';
 
-const AverageWaveformsView: FunctionComponent<SortingViewProps> = (props) => {
+export type AverageWaveformAction = ActionItem | DividerItem
+
+
+const TOOLBAR_INITIAL_WIDTH = 55
```

Comment:
One of the few 'magic numbers' in here; a 55-pixel width happens to look good on my screen. By using a Slider, we make sure that the user can modify it if needed.

Note that the Toolbar is not vertically centered. This looks a little odd (once you notice it) for the average waveform view. However, since they share a scroll bar, there's not really a way to center it vertically. (Presumably solving that layout issue will be higher-yield when we're in a standardized toolbar component regime.)